powerbi-docs/enterprise/service-premium-interactive-background-operations.md

```diff
@@ -34,6 +34,7 @@ The table below lists the Power BI operations. It provides a short description f
 | [Dataflow refresh](../transform-model/dataflows/dataflows-understand-optimize-refresh.md) | An on demand or scheduled background dataflow refresh, performed by the service or with REST APIs | Dataflows | Background |
 | Dataset on-demand refresh | A background dataset refresh initiated by the user, using the service, REST APIs or public XMLA endpoints | Datasets | Background |
 | Dataset scheduled refresh | A scheduled background dataset refresh, performed by the service, REST APIs or public XMLA endpoints | Datasets | Background |
+| Full Report Email Subscription | A full exported copy of a Power BI report attached to an [email subscription](../consumer/end-user-subscribe.md) | Report | Background |
```

Comment:
@mibruhje, is this a new feature? Why not just 'email subscription'? Also, what do you mean when you say 'A full exported copy' - is this a PBIX?